pkgs/by-name/la/labelle/package.nix

```diff
@@ -8,25 +8,27 @@
 }:
 python3Packages.buildPythonApplication rec {
   pname = "labelle";
-  version = "1.2.3";
+  version = "1.3.2";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "labelle-org";
     repo = "labelle";
     rev = "refs/tags/v${version}";
-    hash = "sha256-nGWffQAXlnKiWBUnSo/IKGdWsLdwLr9N4jAURcVGfj8=";
+    hash = "sha256-+aRq+zwnoHy+kXerFCw1b46OHmWDDpyOiBRu4IvYAQg=";
   };
 
   postPatch = ''
     sed -i 's/hatch-vcs >=0.3.0,<0.4/hatch-vcs >=0.3.0/' pyproject.toml
+    sed -i 's/Pillow>=8.1.2,<11/Pillow>=8.1.2/' pyproject.toml
```

Comment:
please replace these with `pythonRelaxDeps`

Reply:
I had tried that before. However, it doesn't seem to work.

```
Creating a wheel...
* Getting build dependencies for wheel...

ERROR Missing dependencies:
        hatch-vcs<0.4,>=0.3.0
error: builder for '/nix/store/6q5m91l5pcisva8l7ng8hm03wyir5fak-labelle-1.3.2.drv' failed with exit code 1;
```

Here is what I tried:
```
  #postPatch = ''
  #  sed -i 's/hatch-vcs >=0.3.0,<0.4/hatch-vcs >=0.3.0/' pyproject.toml
  #  sed -i 's/Pillow>=8.1.2,<11/Pillow>=8.1.2/' pyproject.toml
  #'';

  buildInputs = [ qt6.qtwayland ];

  nativeBuildInputs = [
    python3Packages.pythonRelaxDepsHook
    qt6.wrapQtAppsHook
    python3Packages.hatchling
    python3Packages.hatch-fancy-pypi-readme
    python3Packages.hatch-vcs
    copyDesktopItems
  ];

  pythonRelaxDeps = [
    "hatch-vcs"
    "Pillow"
  ];
```


Any idea?

Reply:
ah, this would require https://github.com/NixOS/nixpkgs/pull/331315

Please use `substituteInPlace pyproject.toml --replace-fail ...` instead

Reply:
Done.